### PR TITLE
Fix Datalab init action on Debian 10

### PR DESCRIPTION
**Pull request by medb**

### CODE REVIEW

File: datalab/datalab.sh [hunk 1]
```diff
@@ -111,20 +111,18 @@ function configure_master() {
...
-${ADD_BACKPORTS}
+
 ADD dataproc.list /etc/apt/sources.list.d/
 ADD trusted.gpg /tmp/vm_trusted.gpg
-
 RUN apt-key add /tmp/vm_trusted.gpg
+
+RUN apt-get update
+RUN apt-get install -y software-properties-common
+RUN add-apt-repository 'deb http://archive.ubuntu.com/ubuntu bionic main'
```
Comment (functicons) [inline datalab/datalab.sh]: Add comment for why we need this.

File: datalab/datalab.sh [hunk 2]
```diff
@@ -111,20 +111,18 @@ function configure_master() {
...
   cat <<EOF >Dockerfile
 FROM ${DOCKER_IMAGE}
-${ADD_BACKPORTS}
+
 ADD dataproc.list /etc/apt/sources.list.d/
 ADD trusted.gpg /tmp/vm_trusted.gpg
-
 RUN apt-key add /tmp/vm_trusted.gpg
+
+RUN apt-get update
```
Comment (functicons) [inline datalab/datalab.sh]: Can we add if check to only apply this change to Debian 10? otherwise there is a risk to break users of other OSs.

Comment (medb) [reply datalab/datalab.sh]: We have tests for all the OSes and they are passing, so we shouldn't break existing users.

File: datalab/datalab.sh (hunk 1 above)
Comment (medb) [reply datalab/datalab.sh]: Added.

**Pull request merged**